c/prima.c

```diff
@@ -21,50 +56,127 @@ int lincoa_c(prima_obj calfun, const void *data, const int n, double x[], double
              const double xl[], const double xu[],
              int *nf, const double rhobeg, const double rhoend, const double ftarget, const int maxfun, const int npt, const int iprint, int *info);
 
+int prima_check_options(prima_options * opt, int n)
+{
+  if (!opt)
+    return PRIMA_NULL_OPTIONS;
+  if (!opt->xl)
+  {
+    opt->xl = (double*)malloc(n * sizeof(double));
+    opt->_allocated_xl = 1;
+    for (int i = 0; i< n; ++ i)
+      opt->xl[i] = -HUGE_VAL;
```

Comment:
`-HUGE_VAL` should be replaced by `-INFINITY` here.

Reply:
ok